### PR TITLE
Correction des erreurs/alertes de flake8

### PR DESCRIPTION
**Pull request by orandin**
| Q | R |
| --- | --- |
| Type de modification | Refactoring |
| Ticket(s) (_issue(s)_) concerné(s) |  |

Mon linter Flake8 sur Atom a vu rouge sur le fichier `settings.py`.
Voici toutes les erreurs répertoriées :

``` diff
- Flake8 Warning F401 — 'dirname' imported but unusedat line 5 col 21
- Flake8 Error E261 — at least two spaces before inline commentat line 97 col 70
- Flake8 Error E123 — closing bracket does not match indentation of opening bracket's lineat line 150 col 13
- Flake8 Error E265 — block comment should start with '# 'at line 223 col 9
- Flake8 Error E265 — block comment should start with '# 'at line 230 col 9
- Flake8 Error E402 — module level import not at top of fileat line 369 col 1
- Flake8 Warning F811 — redefinition of unused 'message_constants' from line 8at line 369 col 1
- Flake8 Error E261 — at least two spaces before inline commentat line 552 col 62
- Flake8 Warning F403 — 'from settings_prod import *' used; unable to detect undefined namesat line 575 col 5
```

### CODE REVIEW

File: zds/settings.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 import os
 import sys
-from os.path import dirname
```
Comment (orandin) [inline zds/settings.py:5]: `Flake8 Warning F401 — 'dirname' imported but unusedat line 5 col 21`

File: zds/settings.py [hunk 2]
```diff
@@ -94,7 +93,7 @@
 )
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = 'n!01nl+318#x75_%le8#s0=-*ysw&amp;y49uc#t=*wvi(9hnyii0z' # noqa
+SECRET_KEY = 'n!01nl+318#x75_%le8#s0=-*ysw&amp;y49uc#t=*wvi(9hnyii0z'  # noqa
```
Comment (orandin) [inline zds/settings.py:96]: `Flake8 Error E261 — at least two spaces before inline commentat line 97 col 70`

File: zds/settings.py [hunk 3]
```diff
@@ -147,7 +146,7 @@
                 'zds.utils.context_processor.git_version',
             ],
             'debug': DEBUG,
-            }
+        }
```
Comment (orandin) [inline zds/settings.py:149]: `Flake8 Error E123 — closing bracket does not match indentation of opening bracket's lineat line 150 col 13`

File: zds/settings.py [hunk 4]
```diff
@@ -220,14 +219,14 @@
     ),
     'DEFAULT_PARSER_CLASSES': (
         'rest_framework.parsers.JSONParser',
-        #'rest_framework.parsers.XMLParser',
+        # 'rest_framework.parsers.XMLParser',
```
Comment (orandin) [inline zds/settings.py:222]: `Flake8 Error E265 — block comment should start with '# 'at line 223 col 9`

File: zds/settings.py [hunk 5]
```diff
@@ -220,14 +219,14 @@
...
         'rest_framework_xml.parsers.XMLParser',
         'rest_framework.parsers.FormParser',
         'rest_framework.parsers.MultiPartParser',
     ),
     'DEFAULT_RENDERER_CLASSES': (
         'rest_framework.renderers.JSONRenderer',
-        #'rest_framework.renderers.XMLRenderer',
+        # 'rest_framework.renderers.XMLRenderer',
```
Comment (orandin) [inline zds/settings.py:229]: `Flake8 Error E265 — block comment should start with '# 'at line 230 col 9`

File: zds/settings.py [hunk 6]
```diff
@@ -366,7 +365,6 @@
 # Fake mails (in console)
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-from django.contrib.messages import constants as message_constants
```
Comment (orandin) [inline zds/settings.py:369]: `Flake8 Error E402 — module level import not at top of fileat line 369 col 1`

`Flake8 Warning F811 — redefinition of unused 'message_constants' from line 8at line 369 col 1`
=> doublon avec la ligne 7 de la nouvelle version ou 8 de l'ancienne version.

File: zds/settings.py [hunk 7]
```diff
@@ -549,7 +547,7 @@
 SOCIAL_AUTH_FACEBOOK_KEY = ""
 SOCIAL_AUTH_FACEBOOK_SECRET = ""
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = "696570367703-r6hc7mdd27t1sktdkivpnc5b25i0uip2.apps.googleusercontent.com"
-SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = "mApWNh3stCsYHwsGuWdbZWP8" # noqa
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = "mApWNh3stCsYHwsGuWdbZWP8"  # noqa
```
Comment (orandin) [inline zds/settings.py:550]: `Flake8 Error E261 — at least two spaces before inline commentat line 552 col 62`

File: zds/settings.py [hunk 8]
```diff
@@ -572,6 +570,6 @@
 
 # Load the production settings, overwrite the existing ones if needed
 try:
-    from settings_prod import *
+    from settings_prod import *  # noqa
```
Comment (orandin) [inline zds/settings.py:573]: `Flake8 Warning F403 — 'from settings_prod import *' used; unable to detect undefined namesat line 575 col 5`

**Comment by gustavi** [comment]
À changer : [cette ligne](https://github.com/zestedesavoir/zds-site/blob/20d324eb32eeb45871d2638f2de06049b701a019/tox.ini#L50) et [celle-ci](https://github.com/zestedesavoir/zds-site/blob/20d324eb32eeb45871d2638f2de06049b701a019/doc/source/utils/git-pre-hook.rst#L21).

**Comment by gustavi** [comment]
Parfait, merci

**Pull request merged**